Jenkinsfile

```diff
@@ -21,6 +22,19 @@
 /* groovylint-disable-next-line CompileStatic */
 job_status_internal = [:]
 
+void get_rpm_relval() {
```

Comment:
Seems like this should also take into account the `env.BuildType` value.  If it's set to "release", then DAOS_RELVAL should be "".

Reply:
I don't think that's how it works today.   If you look at the Makefile that builds RPMs, you can see that it is entirely based on whether the file named TAG has been changed in the commit, not whether release build is used or not.   If we want to adjust that, we can but that just seems how it works today.

https://github.com/daos-stack/daos/blob/aaf63c13e7612dcf9e387032bce9d56679067a6d/utils/rpms/Makefile#L18-L20

I don't think we should block this on making that work.

The equivalent code for this PR is here:

https://github.com/daos-stack/daos/blob/33759e9f8479432c811919ffee53c6f694f2ec12/Jenkinsfile#L25-L36

Reply:
You're correct.